zipline/pipeline/factors/technical.py

```diff
@@ -683,3 +679,115 @@ def compute(self, today, assets, out, highs, lows, closes):
             )),
             2
         )
+
+
+class MovingAverageConvergenceDivergence(_ExponentialWeightedFactor):
```

Comment:
Why is this inheriting from `_ExponentialWeightedFactor`  I don't think it will work if you call any of the inherited classmethods.